### PR TITLE
Qa fixes

### PR DESCRIPTION
**Pull request by SofiaMargariti**

### CODE REVIEW

File: tests/test_commands.py [hunk 1]
```diff
@@ -151,17 +151,18 @@ def test_pull_with_branch_no_git_repo(self, project_mock, log_mock, bmock):
...
+        bmock.return_value = 'a-branch'
         pr_instance = MagicMock()
         project_mock.return_value = pr_instance
-        with self.assertRaises(SystemExit):
-            cmd_pull(['--branchname', 'somebranch'], '.')
-        log_mock.error.assert_called_once_with(
-            "--branchname options should be used along with "
-            "the --branch option."
+        cmd_pull(['--branch'], '.')
+        assert pr_instance.pull.call_count == 1
```
Comment (rigaspapas) [inline tests/test_commands.py]: I believe we should avoid plain assertions and use the unittest's ones

Comment (SofiaMargariti) [reply tests/test_commands.py]: I googled this but I couldn't find any strong reason not to use it. Do you have a specific reason why not to?

Comment (dbendilas) [reply tests/test_commands.py]: +1

Comment (dbendilas) [reply tests/test_commands.py]: @SofiaMargariti we are already using unittest's assertions on the same file, so why not be consistent?

Comment (rigaspapas) [reply tests/test_commands.py]: @SofiaMargariti for consistency most importantly, but there is also a real difference as mentioned [here](https://stackoverflow.com/a/2958183)

**Pull request merged**